src/index.html

```diff
@@ -11,7 +11,153 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="
+      https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="form-set">
+        <legend>Personal infomation</legend>
+
+        <label class="form-field">Surname:
+          <input type="text" name="surname">
+        </label>
+
+
+        <label class="form-field"> Name:
+          <input type="text" name="name">
+        </label>
+
+        <label class="form-field">How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1" max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">Full date of birth:
+          <input
+            type="date"
+            name="b-date"
+            min="1900-01-01"
+            max="2023-01-01"
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+
+
+        <label class="form-field">I accept the term of the agreement
+          <input type="checkbox"
+            name="terms"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="form-set">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input type="password">
+        </label>
+      </fieldset>
+
+      <fieldset class="form-set" >
+        <legend>
+          An intersting fact about you!
+        </legend>
+
+        <label class="form-field cats">
+          Do you love cats?
+          <input
+            type="radio"
+            id="Yes"
+            name="cat"
+            value="yes"
+          >
+          Yes
+        </label>
+        <label class="form-field cats">
+          <input
+            type="radio"
+            id="No"
+            name="cat"
+            value="no"
+          >
+          No
+        </label>
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" required>
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+          name="cars"
+          multiple
+          required
+          >
```

Comment:
Please check all places before send to review!
```suggestion
          <select
            name="cars"
            multiple
            required
          >
```